locallib.php

```diff
@@ -2626,6 +2626,32 @@ function theme_boost_union_is_active_theme() {
     }
 }
 
+/**
+ * Helper function to show an alert if Boost Union is not the active theme but someone tries to access
+ * Boost Union's settings or files.
+ */
+function theme_boost_union_is_not_active_alert() {
```

Comment:
Please add some comments to this new function (you have surely noticed that the comment-to-code ratio in Boost Union is quite high).